src/runtime/hexagon_remote/elf.h

```diff
@@ -143,7 +159,8 @@ struct elf_t {
         // stuff, the same size again to make a writeable copy, and
         // then an extra page for the global offset table for PIC
         // code.
-        buf = (char *)mmap(NULL, size * 2 + global_offset_table_size,
+        mmap_ptr = (mmap_ptr_fn) halide_get_symbol("mmap");
```

Comment:
i just rebuilt with 8.0.05 and there are no differences in the libs i pushed up and what i rebuilt

i see invocations of the tools pointing at the 8.0.05 release.